website/test.ts

```diff
@@ -0,0 +1,208 @@
+// Copyright 2018 Ryan Dahl <ry@tinyclouds.org>
+// All rights reserved. MIT License.
+
+import { assertEqual, test } from "liltest";
+import { generateDoc } from "./core";
+import * as util from "./util";
+import "./serializer_function";
+import "./serializer_types";
+import "./serializer_keywords";
+import "./serializer_interface";
+import "./serializer_enum";
+import "./serializer_class";
+import "./serializer_object";
+import "./serializer_declaration";
+import "./serializer_module";
```

Comment:
It would be nice if you didn't have to import all these things- maybe they should be imported in core?
Also, "core.ts" isn't so descriptive. I think "gendoc.ts" or something similar would be easier.

Reply:
will do

Reply:
done